setup.cfg

```diff
@@ -30,6 +30,18 @@ python_requires = >=3.6
 package_dir =
     =.
 
+[options.extras_require]
+pymemcache =
+  pymemcache
+memcached
+  python-memcached
+bmemcached
```

Comment:
aren't `memcached` and `bmemcached` mussing an `=` here?

Reply:
They are. That's what I get for rushing stuff after 6pm 🙈😅 Will fix